subprojects/core/src/main/java/org/gradle/deployment/internal/DeploymentRegistry.java

```diff
@@ -51,8 +49,9 @@
 
     /**
      * Behavior when a deployment is out-of-date.
+     * 
+     * @since 4.2
```

Comment:
Why add `@since` here, especially on an internal API?

Reply:
I removed the `@since`.